src/docs/product/session-replay/getting-started.mdx

```diff
@@ -25,3 +25,28 @@ Session Replay supports all browser-based JavaScript applications. It’s built
 - [Remix](/platforms/javascript/guides/remix/session-replay/)
 - [Svelte](/platforms/javascript/guides/svelte/session-replay/)
 - [Vue](/platforms/javascript/guides/vue/session-replay/)
+
+## Backend Error Linking
```

Comment:
I'm not about about calling Backend Error Linking because basically this is Tracing we're talking about.

If folks have Tracing setup, it should just work.

Reply:
Thoughts on changing it to backend error tracing to make it clearer?

Reply:
How about calling it what the user will see: "Replay in backend errors"

And in the description adjust slightly to mention that for as long as distributed tracing is enabled, backedn errors and transactions will display a replay of what was going on on the frontend?

Reply:
I like that suggestion! Just updated the wording